packages/react/src/hooks/buckets.ts

```diff
@@ -12,19 +12,24 @@ import {
 } from "wagmi";
 
 import { getChain, getObjectApiUrl } from "@recallnet/chains";
-import { bucketManagerAbi, bucketManagerAddress } from "@recallnet/contracts";
+import {
+  iBucketFacadeAbi,
+  iMachineFacadeAbi,
+  iMachineFacadeAddress,
+} from "@recallnet/contracts";
+import { base32ToHex, hexToBase32 } from "@recallnet/fvm/utils";
 
 export function useListBuckets(owner?: Address) {
   const chainId = useChainId();
   const { address } = useAccount();
   const contractAddress =
-    bucketManagerAddress[chainId as keyof typeof bucketManagerAddress];
+    iMachineFacadeAddress[chainId as keyof typeof iMachineFacadeAddress];
 
   const arg = owner ?? address;
 
   return useReadContract({
     address: contractAddress,
-    abi: bucketManagerAbi,
+    abi: iMachineFacadeAbi,
```

Comment:
example: to list buckets, we use the `iMachineFacadeAbi` interface and hit the machine/ADM actor. but down below, we use the `iBucketFacadeAbi` interface and directly hit the bucket contract